staging/src/k8s.io/cli-runtime/pkg/genericclioptions/json_yaml_flags.go

```diff
@@ -59,7 +64,9 @@ func (f *JSONYamlPrintFlags) ToPrinter(outputFormat string) (printers.ResourcePr
 
 // AddFlags receives a *cobra.Command reference and binds
 // flags related to JSON or Yaml printing to it
-func (f *JSONYamlPrintFlags) AddFlags(c *cobra.Command) {}
+func (f *JSONYamlPrintFlags) AddFlags(c *cobra.Command) {
+	c.Flags().BoolVar(&f.stripManagedFields, "strip-managed-fields", false, "Strip the managed fields from object to make it less verbose")
```

Comment:
I'd possibly name this variable `show-managed-fields`, and default it to false. That seems to be the general consensus so far.

Reply:
I have updated the flag as you said.